README.md

```diff
@@ -100,7 +104,7 @@ Select 1:
 What is the difference between a fork and a clone?
 
 ```text
-Your answer here.
+You need to fork to do a pull request and have your changes submitted/merged into the original. You must then clone it to your local computer to make changes. You can clone any repository and make changes, but you cannot submit a pull request.
 ```
```

Comment:
While all this is true for the current GA x student fork set up, this won't necessarily be the case in the real world. You may find that you have write access to an upstream repo, in which case you wouldn't need to have a fork in order to submit a PR and make changes. That being said, forking-and-cloning is a common workflow set-up, even in those situations.